docs/samples/README.md

```diff
@@ -17,21 +17,25 @@ After models are deployed onto model servers with KFServing, you get all the fol
+| Out-of-the-box Predictor  | Exported model| Prediction Protocol | HTTP | gRPC | Versions| Examples |
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+| [SKLearn KFServer](https://github.com/kubeflow/kfserving/tree/master/python/sklearnserver) | [Pickled Model](https://scikit-learn.org/stable/modules/model_persistence.html) | v1 | :heavy_check_mark: | -- | 0.20.3 | [SKLearn Iris](./v1alpha2/sklearn)  |
+| [SKLearn MLServer](https://github.com/SeldonIO/MLServer) | [Pickled Model](https://scikit-learn.org/stable/modules/model_persistence.html) | v2 | :heavy_check_mark: | :heavy_check_mark: | 0.23.1 | [SKLearn Iris V2](./v1beta1/sklearn)  |
+| [XGBoost KFServer](https://github.com/kubeflow/kfserving/tree/master/python/xgbserver) | [Saved Model](https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html) | v1 | :heavy_check_mark: | -- | 0.82 | [XGBoost Iris](./v1alpha2/xgboost)  |
+| [XGBoost MLServer](https://github.com/SeldonIO/MLServer) | [Saved Model](https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html) | v2 | :heavy_check_mark: | :heavy_check_mark: | 1.1.1 | [XGBoost Iris V2](./v1beta1/xgboost)  |
+| [PyTorch KFServer](https://github.com/kubeflow/kfserving/tree/master/python/pytorchserver) | [Eager Model](https://pytorch.org/docs/master/generated/torch.save.html) | v1 | :heavy_check_mark: | -- | 1.3.1 |  [PyTorch Cifar10](./v1alpha2/pytorch)  |
+| [PMML KFServer](https://github.com/kubeflow/kfserving/tree/master/python/pmmlserver) | [PMML](http://dmg.org/pmml/v4-4-1/GeneralStructure.html) | v1 | :heavy_check_mark: | -- | [PMML4.4.1](https://github.com/autodeployai/pypmml) | [SKLearn PMML](./v1alpha2/pmml)  |
+| [TorchServe](https://pytorch.org/serve/server.html) | [Eager Model/TorchScript](https://pytorch.org/docs/master/generated/torch.save.html) | v1 | :heavy_check_mark: | WIP | | [TorchServe Examples](./custom/torchserve)  |
+| [TFServing](https://www.tensorflow.org/tfx/guide/serving) | [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model) | v1 | :heavy_check_mark: | :heavy_check_mark: | [TFServing Versions](https://github.com/tensorflow/serving/releases) | [TensorFlow Flowers](./v1alpha2/tensorflow)  |
+| [ONNXRuntime](https://github.com/microsoft/onnxruntime)  | [Exported ONNX Model](https://github.com/onnx/tutorials#converting-to-onnx-format) | v1 | :heavy_check_mark: | :heavy_check_mark: | [Compatibility](https://github.com/microsoft/onnxruntime#compatibility) |[ONNX Style Model](./v1alpha2/onnx)  |
+| [Triton Inference Server](https://github.com/triton-inference-server/server) | [TensorFlow,TorchScript,ONNX,TensorRT](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/model_repository.html)| v2 | :heavy_check_mark: | :heavy_check_mark: | [Compatibility Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html)| [TorchScript Cifar10](./v1beta1/triton/torchscript) |
```

Comment:
Maybe worth moving Triton up, given its more popular and also compatible with v2 API